src/Microsoft.DotNet.Metrics/Microsoft.DotNet.Metrics.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
+    <TargetLatestRuntimePatch>False</TargetLatestRuntimePatch>
+    <SignAssembly>false</SignAssembly>
```

Comment:
We also probably should sign this assembly. The only reason we don't sign some of the assemblies in arcade-services is because Octokit isn't signed.